react-app/src/features/package/package-activity/index.tsx

```diff
@@ -48,7 +48,7 @@ const Submission = ({ packageActivity }: SubmissionProps) => {
     <div className="flex flex-col gap-6">
       <div>
         <h2 className="font-bold text-lg mb-2">Attachments</h2>
-        {attachments.length > 0 ? (
+        {attachments?.length > 0 ? (
```

Comment:
I'm not seeing this error in my environment since `attachments` is given a default `[]` value. Is it due to your TS extension?